Resources/Maps/Test/dev_map.yml

```
@@ -5070,7 +5070,7 @@ entities:
     - type: Transform
       pos: 3.5,7.5
       parent: 179
-- proto: SpawnMobMouse
+- proto: SpawnMobCorgi
   entities:
   - uid: 1050
     components:
```

```suggestion
    components:
    - type: MetaData
      desc: I swore it was small and gray?
      name: mouse
```

Maybe you can think of a better description but it would be fun if we paid homage to the old mouse (You will NOT be missed 😆). Also not sure if that is the totally correct way of adding the custom name and stuff.

---

better idea i think add a runtime renamed to tom
then rename mouse to jerry and change to mouse2

---

Nooooo 😢....

---

2 extra prototypes well worth it